Partial fraction decomposition

ozr0012 opened the pull request:
# New content

<!--- Briefly describe the changes made. -->

<!--- Replace XXX with any associated issues. --->

Resolves #38

## Checklist

- [ ] New pages are added to the table of contents.
- [ ] Everything displays correctly in the web-generated preview.
- [ ] Appropriate attribution has been given to source materials.
- [ ] My name is included alphabetically in `credits.md`.

mzbush reviewed (changes requested):
This is a good first pass! It does a very good job at following the notes. I just have a few formatting suggestions for the equations to make them look better.

mphoward commented:
@ozr0012 please leave a comment here when you have addressed @mzbush's comments so I can give it a final review!

ozr0012 reviewed (commented):
- can't figure out how to make the "topic" directive look right in the preview for line 48 where it says " Partial Fraction Decomposition" 
- also the \arctan x in line 121 is showing up red in the preview for some reason although I tried to do the \arctanx like was suggested

ozr0012 commented on calculus/integration-partial-fractions.md:
```diff
@@ -1 +1,75 @@
+
+$ax^2 + bx + c \to \frac{Ax+B}{ax^2+bx+c}$
+
+$(ax^2 + bx+c)^k $: repeat like for linear factor
```
tried to add topic directive but looks weird in Preview so I don't think I did it right

ozr0012 replied:
also for the \arctanx in line 121, when I added the \ in front like suggested it shows up as red in the preview

mzbush replied:
In the `nonlinear-equations/root-finding.md` there's a good example of how to use the `{topic}` and `{example}` directives if you're struggling to get them working correctly.

The reason that `\arctanx` doesn't work is because it thinks the x at the end is part of the word when it's not supposed to be. It should be `\arctan(x)`

mphoward replied:
`\arctan x` should also work if you don't want parentheses!

mzbush commented on calculus/integration-partial-fractions.md:
```diff
@@ -1 +1,134 @@
+
+ 
+
+\begin{align}
+3x+11 &= A_1(x+2) + A_2(x-3) \\
+&= (A_1 + A_2)x + (2A_1 - 3A_2) \\
+\end{align}
+
+\begin{align}
+ A_1  + A_2 = 3
+\end{align}
+
+\begin{align}
+2A_1 - 3A_2 = 11
+\end{align}
+
+\begin{bmatrix} 1 & 1 & 3\\ 2 & -3 & 11 \end{bmatrix} \to \begin{bmatrix} 1 & 1 & 3\\ 0 & -5 & 5 \end{bmatrix} \to \begin{bmatrix} 1 & 1 & 3\\ 0 & 1 & -1 \end{bmatrix} \to \begin{bmatrix} 1 & 0 & 4\\ 0 & 1 & -1 \end{bmatrix}
+
+\begin{align}
+A_1=4  A_2=-1
+\end{align}
+
+\begin{align}  
+y&=\int\left( \frac{4}{x-3}-\frac{1}{x+2} \right)dx \\ 
+&= 4\ln| x-3 | - \ln|x+2| + C  
+\end{align}
```
The example could use a few words in-between the sequences of equations that just explain what is happening for the uninitiated audience.

ozr0012 replied:
added some words in between so let me know if i need to adjust/add more

mzbush replied:
The first example could use more use more words, especially the method used to solve the linear system of equations.

mphoward replied:
I agree with @mzbush. I would add a couple lines in between these equations (like you did below) explaining the steps: cross-multiply, group like powers of x and equate coefficients, solve the linear system of equations.

mzbush commented on calculus/integration-partial-fractions.md:
```diff
@@ -1 +1,134 @@
+&= (A_1 + A_2)x + (2A_1 - 3A_2) \\
+\end{align}
+
+\begin{align}
+ A_1  + A_2 = 3
+\end{align}
+
+\begin{align}
+2A_1 - 3A_2 = 11
+\end{align}
```
```suggestion
\begin{align}
 A_1  + A_2 &= 3 \\
 2A_1 - 3A_2 &= 11
\end{align}
```

These two equations make up a single linear system of equations that gets solved in the line below, so they should be grouped together.

mzbush commented on calculus/integration-partial-fractions.md:
```diff
@@ -1 +1,134 @@
+\end{align}
+
+\begin{align}
+\left(ax^2 + bx+c \right)^k:repeat like for linear factor
```
```suggestion
\left(ax^2 + bx+c \right)^k \text{: repeat like for linear factors}
```
Use `\text` to get text inside an equation to be treated like text instead of part of an expression.

mzbush commented on calculus/integration-partial-fractions.md:
```diff
@@ -1 +1,134 @@
+\end{equation}
+
+
+\begin{equation}
+A_1+A_3 = 0
+\end{equation}
+
+
+\begin{equation}
+-4A_1 + A_2-8A_3 + B _3 =1
+\end{equation}
+
+
+\begin{equation}
+3A_1 + 16A_3 - 8B_3 = -29
+\end{equation}
+
+
+\begin{equation}
+-12A_1+3A_2+16B_3 =5
+\end{equation}
```
```suggestion
\begin{align}
A_1+A_3 &= 0 \\
-4A_1 + A_2-8A_3 + B_3 &=1 \\
3A_1 + 16A_3 - 8B_3 &= -29 \\
-12A_1+3A_2+16B_3 &=5 \\
\end{align}
```

Like with the above example, these should all be grouped together.

mzbush commented on calculus/integration-partial-fractions.md:
```diff
@@ -1 +1,134 @@
+
+
+\begin{equation}
+= \ln|x-4|+\frac{5}{x-4} -\frac{1}{2}ln (x^2 + 3)+\frac{2}{\sqrt{3}} \arctan(\frac{x}{\sqrt{3}}) + C
```
```suggestion
= \ln|x-4|+\frac{5}{x-4} -\frac{1}{2}ln (x^2 + 3)+\frac{2}{\sqrt{3}} \arctan\left( \frac{x}{\sqrt{3}} \right) + C
```

mzbush reviewed (changes requested):
This pull request is looking much better now! If you're having trouble with the directives like `{topic}` and `{example}`, I suggest looking at 'nonlinear-equations/root-finding.md' in the main branch to see how it's done there.

ozr0012 commented:
added more commentary but for some reason it says that I have a typo :equition in there that is messing a couple of lines up at the end. I cannot find the typo and thought i fixed them all so it could not be updating.

mphoward reviewed (commented):
Thank you! I addressed the code formatting issues (you were missing some equation blocks). I also tried to help clean up the text you added. Can you please go over everything a final time, and then re-request a review so I can merge? This looks good to go from my side after you do that!

mzbush commented on calculus/integration-partial-fractions.md:
```diff
@@ -1,5 +1,154 @@
+\end{align}
+
+This is a system of linear equations that can be solved, e.g., using a matrix
+technique. Form the augemented matrix then perform Gauss-Jordan elimination:
```
```suggestion
technique. Form the augmented matrix then perform Gauss-Jordan elimination:
```

mzbush commented on calculus/integration-partial-fractions.md:
```diff
@@ -1,5 +1,154 @@
+= \begin{bmatrix} 0\\ 1\\ -29 \\ 5 \end{bmatrix}
+\end{equation}
+
+solving numerically gives $A_1 = 1$, $A_2 = -5$, $A_3 = -1$, and $B_3 =2$.
```
```suggestion
Solving numerically gives $A_1 = 1$, $A_2 = -5$, $A_3 = -1$, and $B_3 =2$.
```

mzbush reviewed (changes requested):
This pull request looks great now and much easier to follow. I have only two suggestions and after those are dealt with, I see nothing that should prevent this from being merged.
